### PR TITLE
[Author] [New] add Meshki v0.5.1 with npm auto-update

### PR DESCRIPTION
**Pull request by Borderliner**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Borderliner/Meshki
 * Official website (optional, not the repository): http://meshki.borderliner.ir
 * NPM package url (optional): https://npmjs.com/package/meshki
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 5
   - Count of watchers: 5
   - Count of forks: 0
 * NPM download stats (optional):
   - Downloads in the last day: 24
   - Downloads in the last week: 379
   - Downloads in the last month: 642

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

Well the project is pretty new, but it has decent downloads from NPM. Thanks in advance.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@Borderliner Thanks. 
@pvnr0082t @naoh8103 @x09336 would you guys please take a look?

**Comment by pvnr0082t** [comment]
@Borderliner You can commit files under `dist` folder and put them into `0.5.1` folder, or remove the version info in `package.json` file to pass test.

**Comment by Borderliner** [comment]
@PeterDaveHello Well it has passed the tests. What do I need to do now?

File: ajax/libs/meshki/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "meshki",
+  "filename": "meshki.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/meshki/package.json]: I only see `meshki.min.css` under `dist`, not `meshki.min.js`, is this correct?

Comment (pvnr0082t) [inline ajax/libs/meshki/package.json]: But it looks like this of mine:
```json
dist/
├── fonts
│   └── Ubuntu-Regular.ttf
├── meshki.css
├── meshki.js
├── meshki.min.css
└── meshki.min.js
```
anything wrong?

Comment (PeterDaveHello) [inline ajax/libs/meshki/package.json]: hmmmm ... seems my last download or extraction had the problem, looks good now, thanks.

**Comment by PeterDaveHello** [comment]
@pvnr0082t you can try to improve the title next time!

**Comment by PeterDaveHello** [comment]
Okay, I got the problem, v1.0.0/1.1.0 only have `meshki.min.css`, @Borderliner do you have any suggestion?

**Comment by Borderliner** [comment]
The git log was messed up prior to 0.2.1 I guess. Also, all of the files in the `dist` folder are required. Should I add them all in `filename`?
The `fonts` folder is not necessary, but it's good to have it there.

**Comment by PeterDaveHello** [comment]
@Borderliner Filename only point to the single main files, so no need to add the other files.

**Comment by Borderliner** [comment]
@PeterDaveHello So does this mean all my files are accessible through cdnjs now? It only shows `meshki.min.js` file available on https://cdnjs.cloudflare.com

**Comment by PeterDaveHello** [comment]
@Borderliner Nope they are all working, aren't they?  And what about version v1.x?

**Comment by Borderliner** [comment]
No need for v1.x. Thanks for the great work. Another question. Will the CDN be automatically updated when I push a new version (with proper tag)?

**Comment by PeterDaveHello** [comment]
@PeterDaveHello Yes, auto update, but, won't you release v1.x in future?  Shall I mask all the v1.x?

**Comment by Borderliner** [comment]
That's a good point! :smile: I don't know what to do. Do you have any suggestions? I will release v1.x someday and I don't wanna ignore it completely. Would you mind helping me please?

**Comment by PeterDaveHello** [comment]
@Borderliner So what are the current v1.X versions for? Maybe you can just remove them from npm(i am not sure npm allow it)?

**Comment by Borderliner** [comment]
@PeterDaveHello The v1.x versions were old, I changed the versioning system later. It seems that npm does not allow unpublishing packages older than 24 ours. I have to contact them to see if they can remove my old v1.x versions. Thanks a lot.

**Comment by PeterDaveHello** [comment]
@Borderliner Nice.

**Comment by PeterDaveHello** [comment]
@Borderliner is there any updates? Thanks.

**Comment by Borderliner** [comment]
@PeterDaveHello Yes, the new v0.5.2. How come?

**Comment by PeterDaveHello** [comment]
I meant the v1.x, any idea?

**Comment by Borderliner** [comment]
@PeterDaveHello The next version is gonna be v1.1.1. Would this fix the issues with versioning? npm has removed v1.0 and v1.1, but I cannot use these tags again.

**Comment by PeterDaveHello** [comment]
@Borderliner so can I just mask v1.0 and v1.1 on CDNJS? That's much easier, thanks.

**Comment by Borderliner** [comment]
@PeterDaveHello Yes. They're not useful anymore. Thanks a lot for your efforts, and sorry I'm not familiar with CDNs and how they work.

**Comment by PeterDaveHello** [comment]
@Borderliner No worry, I'm going to fix it now.

**Comment by Borderliner** [comment]
I released Meshki v1.2.0, but CDNJS doesn't seem to update it to this version. Also, the info on CDNJS about Meshki is outdated. I've already published Meshki on both bower and npm. Could you please check what's going on? Thanks!

**Comment by PeterDaveHello** [comment]
Thanks @Borderliner, I'll take a look at it.

**Comment by PeterDaveHello** [comment]
Hello @Borderliner, thanks for following up, I found that you added one more font file in the v1.2.0, since we don't have AI on the bot, once the structure changed, we need a human to review/audit the change, so there will be a delay, sorry for the inconvenience, I'm now adding v1.2.0, thanks for your report.

**Comment by Borderliner** [comment]
Actually I didn't intend to "add" another font file, the Ubuntu font file slipped through my hands and I accidentally pushed it into the new version. But that's alright, I will removed it in the next minor version. Thanks!

**Comment by PeterDaveHello** [comment]
Okay @Borderliner, so just let you know, the next release also needs a human review which means will be delayed for few hours.

**Comment by Borderliner** [comment]
@PeterDaveHello Sorry to bother you again, but I've released a new version, and the extra font is removed now. Could you please review it?

**Comment by PeterDaveHello** [comment]
@Borderliner Done.

**Pull request merged**